react-app/nginx.conf

```diff
@@ -0,0 +1,23 @@
+server {
+  listen 80;
+
+  gzip on;
+  gzip_proxied any;
+  gzip_min_length 1000;
+  gzip_types
+      application/json
+      text/plain
+      text/css
+      text/js
+      text/xml
+      text/javascript
+      application/javascript
+      application/x-javascript
+      image/svg+xml;
+
+  location / {
+    root   /usr/share/nginx/html;
+    index  index.html index.htm;
+    try_files $uri $uri/ /index.html;
+  }
+}
```

Comment:
will we need to serve `config.js`, `robots.txt` and a ping endpoint?
ref https://github.com/oursky/project-house/blob/939893fdc86f94cf7a05fab77e830740c264375a/web/nginx.conf#L63

Reply:
I think I will handle this in the deployment pr